components/ItemCard.js

```diff
@@ -0,0 +1,63 @@
+import styled from 'styled-components';
+
+export default function ItemCard({ name, date, quantity }) {
+  return (
+    <StyledItem>
+      <StyledName>{name}</StyledName>
+      <StyledQty>Qty:</StyledQty>
+      <StyledQtyValue>{quantity}</StyledQtyValue>
+      <StyledDate>date:</StyledDate>
+      <StyledDateValue>{date}</StyledDateValue>
+    </StyledItem>
+  );
+}
+
+const StyledItem = styled.li`
+  border: 1px solid black;
+  margin: 1%;
+  margin-left: 0;
```

Comment:
Hey Neele,
margin ist hier doppelt angegeben bzw. margin-left ist in margin bereits integriert. Du könntest es auch so schreiben: 
`margin: 1% 1% 1% 0` < es ist zu lesen wie eine Uhr - oben rechts unten links :)